lua/core/menu/home.lua

```diff
@@ -3,6 +3,8 @@ local m = {
   list = {"SELECT", "SYSTEM", "SLEEP"}
 }
 
+local p = require 'core/menu/params'
```

Comment:
`params` is globally imported i don't think you need to do this?

Reply:
ah, excellent -- couldn't grok how to get the params reset in there